demisto_sdk/commands/lint/linter.py

```diff
@@ -181,6 +190,23 @@ def __init__(
                     "Unable to find yml file in package"
                 )
 
+    def should_disable_network(self) -> bool:
+        pack_name = get_pack_name(str(self._pack_abs_dir))
+        _pack_ignore_file_path = Path(
+            f"Packs/{pack_name}/{PACKS_PACK_IGNORE_FILE_NAME}"
+        )
+        if _pack_ignore_file_path.exists():
+            config = ConfigParser(allow_no_value=True)
+            config.read(_pack_ignore_file_path)
+            if TESTS_REQUIRE_NETWORK_PACK_IGNORE in config.sections():
```

Comment:
Highly reccomend to refactor this section with the code that does the same thing already in tools and validate_manager.

Reply:
what do you exactly suggests?

Reply:
we should make a function that takes 

_pack_ignore_file_path = Path(
            f"Packs/{pack_name}/{PACKS_PACK_IGNORE_FILE_NAME}"
        )
        if _pack_ignore_file_path.exists():
            config = ConfigParser(allow_no_value=True)
            config.read(_pack_ignore_file_path)
            if TESTS_REQUIRE_NETWORK_PACK_IGNORE in config.sections():
            
            and just returns sections as a dict so it can be reused

Reply:
applied your suggested changed